compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -61,6 +62,21 @@ struct WgpDetails {
 struct ChipDetails {
   uint32_t wgpCount;
   std::optional<StringRef> sku;
+  // Aggregate chip-level bandwidth in TB/s
```

Comment:
```suggestion
  // Aggregate chip-level bandwidth in TB/s.
```
See https://llvm.org/docs/CodingStandards.html#commenting